build.ps1

```diff
@@ -343,6 +343,8 @@ if ($tmpBinaryLog) {
 # Capture MSBuild crash logs
 $env:MSBUILDDEBUGPATH = $LogDir
 
+$env:DOTNET_ROOT = "$PSScriptRoot\.dotnet"
```

Comment:
Shouldn't this special case the x86 case and set `$env:DOTNET_HOME` too?

Reply:
I'm pretty sure we prefer to use DOTNET_ROOT over DOTNET_HOME.

Reply:
Actually, in activate.ps1, we set `${env:DOTNET_ROOT(x86)} = "$PSScriptRoot\.dotnet\x86"`. I think I prefer using that instead?

Reply:
Might work though I don't remember where that variable is read (it's used in our own code).